packages/lake/src/components/LakeSelect.tsx

```diff
@@ -148,32 +157,37 @@ export type SelectProps<V> = {
   style?: StyleProp<ViewStyle>;
 };
 
-export function LakeSelect<V>({
-  title,
-  items,
-  valueStyle,
-  size,
-  color = "current",
-  disabled = false,
-  mode = "normal",
-  placeholder,
-  readOnly = false,
-  id,
-  matchReferenceWidth = true,
-  value,
-  error,
-  hideErrors = false,
-  icon,
-  onValueChange,
-  PopoverFooter,
-  style,
-}: SelectProps<V>) {
+const LakeSelectWithRef = <V,>(
+  {
+    title,
+    items,
+    valueStyle,
+    size,
+    color = "current",
+    disabled = false,
+    mode = "normal",
+    placeholder,
+    readOnly = false,
+    id,
+    matchReferenceWidth = true,
+    value,
+    error,
+    hideErrors = false,
+    icon,
+    onValueChange,
+    PopoverFooter,
+    style,
+  }: SelectProps<V>,
+  ref: ForwardedRef<View>,
```

Comment:
nit: called them `forwardedRef`, it will be easier when we analyze the code (same for `LakeCombobox`, you can rename `externalRef` to `forwarededRef`)